src/cli/CommandsLogic.ts

```diff
@@ -239,7 +277,10 @@ export default class CommandsLogic {
 
     const sInstance = await this.getContractInstance(StakeManager, {}, deployOptions.stakeManagerAddress, Object.assign({}, options), deployOptions.skipConfirmation)
     const pInstance = await this.getContractInstance(Penalizer, {}, deployOptions.penalizerAddress, Object.assign({}, options), deployOptions.skipConfirmation)
+
     const fInstance = await this.getContractInstance(Forwarder, {}, deployOptions.forwarderAddress, Object.assign({}, options), deployOptions.skipConfirmation)
+    await registerForwarderForGsn(fInstance, options)
```

Comment:
**This call is currently broken**
(not sure if related to this place)
it SUCEEDED - but any contract deploy afterwards generate a BUGGY request - with wrong destination